.changeset/shiny-forks-pull.md

```diff
@@ -0,0 +1,6 @@
+---
+'@firebase/firestore': minor
+'@firebase/firestore-types': minor
```

Comment:
Can you add `firebase` to minor bump the main package?

Reply:
done.